index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Anton&family=Crete+Round&family=Inter:wght@400;800&family=Montserrat:wght@700&family=Poppins:wght@200&family=Quicksand:wght@700&family=Roboto:wght@700&family=Space+Mono:ital@1&display=swap');
+    </style>
+  <title>Bootstrap</title>
+</head>
+<body>
+  <header>
+    <nav class="navbar navbar-expand-sm">
+      <div class="container-fluid justify-content-center">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link" href="#">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+    <div><img src="desktop-images/Header-llustration-desktop.svg"  class="background-image" alt=""></div>
+    
+  </header>
+  <section class="first-layout">
+    <div class="container col-sm-5 mt-5">
+      <div class="hey-there">
+        <h1>Hey There. I'm Have <br> <span>I'm a software developer</span></h1>
+      </div>
```

Comment:
I would recommend to add a class to the span item  in order to fit better to the template